description.md

```diff
@@ -17,6 +17,121 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Quizzes
 
+#### Definition
+
+Quizzes are questions that verify specific information and must be verifiable automatically.
+Therefore, multiple-choice questions are regarded as quizzes, while open-ended ones are not.
+Quizzes may be part of practical sessions, lectures or standalone as forms of evaluation.
+
+#### Purpose
+
+Quizzes provide a means to continuously evaluate students.
+This creates a positive feedback loop between trainers and trainees whereby both can track the students' progress in real time.
+
+Furthermore, quizzes serve as practice items for students to solve at their own pace.
+They may also be used for evaluation items that contribute to the students' overall grade.
+
+Another desired aspect is that quizzes be easily extractable and convertible between formats, as described [below](#deployment), and repurposed by other trainers.
+
+Despite all of their strengths, quizzes also come with a few shortcomings.
+One such downside is the binary nature of the feedback loop.
+Because multiple-choice answers may be either correct or wrong, there is little room for nuance regarding the feedback that students receive upon answering incorrectly.
+Similarly, quizzes cannot be easily tailored and personalized for the individual needs of each student.
+In addition, non-technical fields of study may find little benefit in quizzes as they rely more on arguments essays rather than concrete, technical information.
+
+#### Deployment
+
+Each trainer uses a specific teaching rendering platform.
+Most platforms support quizzes in various formats, such as whether they are graded or ungraded, whether the answers are multiple choice or text boxes, etc.
+Out of the existing teaching platforms, a few highlights are:
+
+- [Moodle](https://moodle.org/) is a free open-source learning platform dedicated to course management.
+It supports graded assignments, quizzes, storing teaching materials and many other educational activities.
```

Comment:
```suggestion
  It supports graded assignments, quizzes, storing teaching materials and many other educational activities.
```

Reply:
What's the effect of this? I see no difference in the rendered output. Is it just to align this sentence with the one above?

Reply:
There is no difference, it produces nesting if you also add an empty line before.